src/main/java/sirbu/ZipCodeTest.java

```diff
@@ -56,24 +57,36 @@ public void testGetZipCodes() {
     @Issue("Zip Code")
     @Step("Add new zip codes")
     public void testPostZipCodes() {
+        List<String> availableZipCodes = given()
+                .header("Authorization", "Bearer " + AuthProvider.getInstance().getReadToken())
+                .get(zipCodeClient.getZipCodesUrl())
+                .jsonPath()
+                .getList("");
```

Comment:
Same there. It is better to move logic to ZipCodeClient getZipCodes() method. 
Test shouldn't know anything about tokens and how to build correct request. It should just reuse method that do it